lib/grammars.coffee

```diff
@@ -497,6 +497,11 @@ module.exports =
     "File Based":
       command: "powershell"
       args: (context) -> [context.filepath.replace /\ /g, "` "]
+      
+  Processing:
+    "File Based":
+      command: "bash"
+      args: (context) -> ['-c', 'processing-java --sketch='+context.filepath.replace("/"+context.filename,"")+' --run']
```

Comment:
why do you replase slashes?
I can't run it on win with this setting.
Getting an `path-to-atom-script-repo-examplesbouncebounce.pde does not exist.`